Source/DiabloUI/selgame.cpp

```diff
@@ -447,9 +447,9 @@ void selgame_Password_Select(int value)
 
 	m_game_data->nDifficulty = nDifficulty;
 	m_game_data->nTickRate = nTickRate;
-	m_game_data->bRunInTown = sgOptions.Gameplay.bRunInTown;
-	m_game_data->bTheoQuest = sgOptions.Gameplay.bTheoQuest;
-	m_game_data->bCowQuest = sgOptions.Gameplay.bCowQuest;
+	m_game_data->bRunInTown = sgOptions.Gameplay.bRunInTown ? 1 : 0;

```

Comment:
Given that `static_cast<uint8_t>()` is a fairly unique concept to C++ I consider it to not be very readable for new comers to the language/code base and so prefer to avoid it when more generic/shorter solutions are available. I suppose ` ? 1 : 0` could be a bit more prone to typos but I think it's acceptable.